src/org/omegat/Bundle.properties

```diff
@@ -441,6 +441,8 @@ TF_OPTIONSMENU_DICTIONARY=Dictionar&y
 TF_OPTIONSMENU_DICTIONARY_FUZZY=Use &Fuzzy Matching for Dictionary Entries
 PREFS_TITLE_DICTIONARY=Dictionary
 PREFS_DICTIONARY_FUZZY=Use &fuzzy matching for dictionary entries
+PREFS_DICTIONARY_AUTO_SEARCH=&Automatically search segment text in dictionary
+TF_MENU_EDIT_SEARCH_DICTIONARY=Search dictionaries
```

Comment:
A minor point, but menu commands like this should be in Title Case.

Reply:
Actually I'm going to fix that for you because I want to test our Azure CI integration.